### ISSUE TITLE
Vesting and PassThrough contracts are not available on app.splits.org

### ISSUE THREAD

**Issue by Inspector-Butters** [0xSplits/docs#63]
I would like to use the Vesting and Passthrough contracts, but they are not available on the https://app.splits.org
it's much simpler to use the UI instead of calling the contracts manually. and I believe there are use cases of these contracts.

**Comment by wminshew** [comment]
thank you for the feedback -- would love to learn more about how you intend to use them. Thank you!

**Comment by Inspector-Butters** [comment]
well, we're building a PGF platform, ["Updraft"](https://github.com/UpdraftFund/), and we want the platform fees to go to humanity, so that no one is excluded from participating because of not having money (so the platform becomes a better public good).

we have identified some platforms that distribute tokens to unique humans. but we are not a 100% sure of their security and we want to initially have a council be in control of distributing the funds to those platforms.

the flow would look something like this:

![updraft-humanities-fees](https://github.com/user-attachments/assets/6b967ebe-b401-4bd6-a93b-fee8a84f988a)

after some time has passed, the passthrough contract would be removed to make things permissionless.

I would also appreciate and welcome any feedback on Updraft.

**Comment by wminshew** [comment]
Thanks -- I see the `PassThrough` contract but am missing where/how `Vesting` is used

**Comment by Inspector-Butters** [comment]
Yes, I forgot about that. the vesting is used to distribute tokens to the development team. (this is in discussion so might not be true in the near future)

**Comment by wminshew** [comment]
@Inspector-Butters which chains are you requesting support for?

**Comment by Inspector-Butters** [comment]
@wminshew I think the ones you have already deployed on are fine. adding them to the UI would be enough. (see [here for pass-through](https://docs.splits.org/core/pass-through#addresses) and [here for vesting](https://docs.splits.org/core/vesting#addresses))

but generally, Arbitrum, Ethereum, and Gnosis would be great to start with.
